schema/to30.xsl

```diff
@@ -37,61 +37,65 @@
 	doctype-public="-//Recordare//DTD MusicXML 3.0 Partwise//EN" />
 
   <!--
-    For the root, only look for score-partwise. Anything else 
+    For the root, only look for score-partwise. Anything else
     as a root element gets ignored.
-  -->  
+  -->
   <xsl:template match="/">
     <xsl:apply-templates select="./score-partwise"/>
   </xsl:template>
 
   <!--
-    Transformations that remove post-3.0 elements and 
+    Transformations that remove post-3.0 elements and
     attributes.
   -->
-  
+
   <!-- Additions in note.mod -->
 
   <!-- Remove inverted-vertical-turn and haydn elements -->
   <xsl:template
     match="inverted-vertical-turn | haydn"/>
-  <!-- 
-    Remove accidental and accidental-mark elements with 
+
+  <!-- Remove soft-accent element -->

```

Comment:
We can probably just combine this with the previous template, and change the comment to something like "Remove new MusicXML 3.1 elements".